bundles/org.openhab.binding.edf/src/main/java/org/openhab/binding/edf/internal/handler/BaseTariffHandler.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.edf.internal.handler;
+
+import static org.openhab.binding.edf.internal.EDFBindingConstants.*;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.edf.internal.dto.BaseTariff;
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.library.unit.CurrencyUnits;
+import org.openhab.core.thing.Thing;
+
+/**
+ * The {@link BaseTariffHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+@NonNullByDefault
+public class BaseTariffHandler extends TariffHandler<BaseTariff> {
+    private final static String DATASET_ID = "c13d05e5-9e55-4d03-bf7e-042a2ade7e49";
```

Comment:
This value is coming from where ?

Reply:
It's the dataset that provide prices on https://www.data.gouv.fr/fr/datasets/r/